### DISCUSSION TITLE
Yo, what link does goo.gl/fd3zc even point to?

### DISCUSSION THREAD

**Discussion by foxsouns** [dnschneid/crouton#4489] [Q&A] [answered]
goo.gl has been dead for like, years. What link does that get sent to at the end? Just wondering.

**Comment by ghost** [comment, marked as answer]
https://raw.githubusercontent.com/dnschneid/crouton/master/installer/crouton

**Reply by foxsouns** [reply]
Awesome haha. Guess I should've guessed something like that.

**Comment by DennisLfromGA** [comment]
@foxsouns ],

> goo.gl has been dead for like, years.

It is not *dead*, the current links still work, you just cannot add any more links to it.

*Hope this helps,*
-**DennisLfromGA**

**Comment by saltedcoffii** [comment]
Have we considered switching to a git.io link?

**Reply by ghost** [reply]
As @DennisLfromGA goo.gl work, you just cannot add more. 
If you wanted a git.io link you could us https://git.io/JZEs0, although it is not the official link.

**Reply by dnschneid** [reply]
Looks like git.io is URL-hash based, not ownership based.  So https://git.io/JZEs0 is as official as it could get...

For now we'll stick with the familiar goo.gl/fd3zc, but if something happens to that link, git.io seems like a good second choice.

**Reply by ghost** [reply]
Might (eventually) need to update the link as it sounds like goo.gl links [will actually end up stop working](https://developers.googleblog.com/en/google-url-shortener-links-will-no-longer-be-available/).

**Reply by dnschneid** [reply]
Yeah, looks like we still have another year (where it'll randomly fail for people curling, unless they add `?si=1`).  Thanks for sufracing the old git.io thread; that seems like the right way to go.

For now I'll have the two links side-by-side so people don't get surprised by the transition.  There's a few other goo.gl links that need to be updated as well.

**Reply by dnschneid** [reply]
....or not?

> Sorry, we're no longer accepting new links to shorten.

Time to find another shortening service...

edit: on second thought, I don't want to have to trust any shortening service not by Google (already trusted because chromebook) or Github (already trusted because that's the link target).  So I'll probably use the git.io link (thanks for making that!!) and then replace the other goo.gl links with their original URLs, even though it's inconvenient.
